_playwright-tests/UI/Templates.spec.ts

```diff
@@ -8,10 +8,27 @@ test.describe('Templates', () => {
   }) => {
     await navigateToTemplates(page);
     await closePopupsIfExist(page);
-
-    const AddButton = page.locator('[data-ouia-component-id="create_content_template"]');
-
-    // Wait for the Add button to become enabled (up to 10 seconds)
-    await AddButton.first().isEnabled({ timeout: 10000 });
+    await page.getByRole('button', { name: 'Add content template' }).click();
+    const architectureFilterButton = page.getByRole('button', { name: 'Select architecture' });
+    await architectureFilterButton.click()
```

Comment:
Do you use `architectureFilterButton` more than once? If not, maybe no need to actually save it in variable.